react/Components/toastArea.tsx

```diff
@@ -8,9 +9,13 @@ import '../styles.global.css'
 import { ShowAlertOptions } from '../utils/showAlertOptions'
 import { toast } from '../utils/definedMessages'
 
-const ToastArea = () => {
-  const DEFAULT_ALERT_AUTOCLOSE_TIMEOUT_MS = 3000
+interface Props {
+  DEFAULT_ALERT_AUTOCLOSE_TIMEOUT_MS: number
```

Comment:
Good! But I think using a property name more simple is better to understand its purpose.

```suggestion
  duration: number
```